app/validation/validator.py

```diff
@@ -1361,6 +1365,22 @@ def _validate_placeholder_transforms(self, transforms, block_id):
 
         return errors
 
+    def _validate_smart_quotes(self, json_schema):
+
+        schema_object = SurveySchema()
+        schema_object.schema = json_schema
+        errors = []
+
+        # pylint: disable=invalid-string-quote
+        quote_regex = re.compile(r"['|\"]+(?![^{]*})+(?![^<]*>)")
+
+        for pointer in schema_object.pointers:
+            schema_text = resolve_pointer(json_schema, pointer)
+            if quote_regex.search(schema_text):
+                errors.append(self._error_message(f'Found dumb quotes(s) in schema text at [{pointer}]'))
```

Comment:
The square braces are a bit confusing here - they result in error messages that appear to be indicating lists.

Reply:
I will remove these.